videomaker/block-template-parts/footer.html

```diff
@@ -1,5 +1,5 @@
-<!-- wp:group {"align":"full","layout":{"type":"flex"},"style":{"spacing":{"padding":{"top":"170px","right":"var(--wp--custom--gap--horizontal)","bottom":"var(--wp--custom--gap--horizontal)","left":"var(--wp--custom--gap--horizontal)"}}}} -->
-<div class="wp-block-group alignfull" style="padding-top: 170px; padding-right: var(--wp--custom--gap--horizontal);padding-bottom: var(--wp--custom--gap--horizontal);padding-left: var(--wp--custom--gap--horizontal);">
+<!-- wp:group {"align":"wide","layout":{"type":"flex"},"style":{"spacing":{"padding":{"top":"170px","right":"var(--wp--custom--gap--horizontal)","bottom":"var(--wp--custom--gap--horizontal)","left":"var(--wp--custom--gap--horizontal)"}}}} -->
+<div class="wp-block-group alignwide" style="padding-top: 170px; padding-right: var(--wp--custom--gap--horizontal);padding-bottom: var(--wp--custom--gap--horizontal);padding-left: var(--wp--custom--gap--horizontal);">
```

Comment:
looks like the paddings are not the same? https://github.com/Automattic/themes/pull/5035 only has top and bottom